ml-agents-envs/mlagents_envs/registry/unity_env_registry.py

```diff
@@ -0,0 +1,125 @@
+from typing import Dict, Iterator, Any, List
+from collections.abc import Mapping
+from mlagents_envs.registry.base_registry_entry import BaseRegistryEntry
+from mlagents_envs.registry.binary_utils import (
+    load_local_manifest,
+    load_remote_manifest,
+)
+from mlagents_envs.registry.remote_registry_entry import RemoteRegistryEntry
+
+
+class UnityEnvRegistry(Mapping):
+    """
+    ### UnityEnvRegistry
+    Provides a library of Unity environments that can be launched without the need
+    of downloading the Unity Editor.
+    The UnityEnvRegistry implements a Map, to access an entry of the Registry, use:
+    ```python
+    registry = UnityEnvRegistry()
+    entry = registry[<environment_identifyier>]
+    ```
+    An entry has the following properties :
+     * `identifier` : Uniquely identifies this environment
+     * `expected_reward` : Corresponds to the reward an agent must obtained for the task
+     to be considered completed.
+     * `description` : A human readable description of the environment.
+
+    To launch a Unity environment from a registry entry, use the `make` method:
+    ```python
+    registry = UnityEnvRegistry()
+    env = registry[<environment_identifyier>].make()
+    ```
+    """
```

Comment:
why is the markdown syntax being used here in the comments? Is it being rendered in the documentation or elsewhere?

Reply:
Some IDE can display markdown in class comments. I wanted to try it out since I am putting some example code in there.